packages/terra-signature/tests/nightwatch/signature-spec.js

```diff
@@ -0,0 +1,20 @@
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+
```

Comment:
This needs updated with the newest release to terra-toolkit v2.0.0. [Here](https://github.com/cerner/terra-core/blob/master/packages/terra-alert/tests/nightwatch/alert-spec.js#L1) is an example of what needs to change.

We will also want to update this to use the expect syntax. With this, the wait for 1000 can be dropped because the newest nightwatch configuration allows for this by deault.